assignment/README.md

```diff
@@ -36,6 +44,11 @@ At a high level, how do you expect people to use your application?
 - What do they gain from your application' use?
 - Are there any website/application examples in the wild to which you can compare your final?
 
+**Our main user is Chief Williams of the city of Louisville, though others in the city could also use this. Our intent is that he will use the site to see where we have predicted fires, and where he can identify locations to help to m mitigate them. He'll be able to click grid cells to see data about that area's risk and information - how many people live there, how many fires there have been in the past, and how high we think its risk is. He can also click some tabs on the side to see the community resource areas, at which he can host outreach efforts for the most impact. We're taking a visual cue from this example from last year, which is similar to our project: **
+
+** https://cdn.rawgit.com/YayinC/Quantify-Bike-Share-Demand-Web-App/master/ ***
```

Comment:
Nice. The sluggishness caused by the size of the hex grid in this application will likely be mitigated by your use of mapbox